graphviz.spec

```diff
@@ -366,11 +395,15 @@ Static development package for %{name}.
 %setup -q
 %apply_patches
 
-autoreconf -fiv
+#cat >./version.m4 <<EOF
+
+#autoreconf -fiv
 
 %build
 export CC=%{__cc}
 export CXX=%{__cxx}
+%{_builddir}/%{name}-%{version}/autogen.sh
```

Comment:
Why not ./autogen.sh instead? Seems simpler and more obvious...

Reply:
Why not indeed. Changed